pkg/nuctl/command/delete.go

```diff
@@ -51,40 +51,56 @@ func newDeleteCommandeer(rootCommandeer *RootCommandeer) *deleteCommandeer {
 
 type deleteFunctionCommandeer struct {
 	*deleteCommandeer
-	functionConfig functionconfig.Config
+	platform.DeleteOptions
 }
 
 func newDeleteFunctionCommandeer(deleteCommandeer *deleteCommandeer) *deleteFunctionCommandeer {
 	commandeer := &deleteFunctionCommandeer{
 		deleteCommandeer: deleteCommandeer,
-		functionConfig:   *functionconfig.NewConfig(),
+		DeleteOptions:    platform.DeleteOptions{},
```

Comment:
Having a new struct for an option that isn't used anywhere other than the command line argument parser is not very clean.

The deleteCommandeer struct may hold the "all" option. Since the underlying DeleteFunctions implementations never examine DeleteOptions.All, having it there is misleading